Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -212,10 +213,10 @@ class WebProcessPool final
     WebProcessProxy* dummyProcessProxy(PAL::SessionID sessionID) const { return m_dummyProcessProxies.get(sessionID).get(); }
 
     void forEachProcessForSession(PAL::SessionID, const Function<void(WebProcessProxy&)>&);
-    template<typename T> void sendToAllProcesses(const T& message);
-    template<typename T> void sendToAllProcessesForSession(const T& message, PAL::SessionID);
+    template<typename T> void sendToAllProcesses(const T& message, ShouldSkipSuspendedProcesses = ShouldSkipSuspendedProcesses::No);
```

Comment:
Do you mind if we don't add the default parameter? Are there just too many call sites to update?
Because frankly, it would be good to force people to think about whether or not the broadcast is useful for suspended processes and have them make a decision.

Reply:
Yeah, there are a lot of callsites. I'll file another bug to track investigate them. This particular one seemed to be triggered the most since it occurs on every tab open/close.